### DISCUSSION TITLE
An error message from the LIDAR Parameters in the IW.dat file

### DISCUSSION THREAD

**Discussion by Hitszh11** [OpenFAST/openfast#2324] [Q&A]
I encountered this error while using the compiled version 3.5.3 of OpenFast and the case in r-test. Later, I tried using version 3.5.0 of OpenFast and the corresponding version of the case, but still encountered this error. I don't know where this error lies, it seems that as long as the parameters in LINDAR Parameters are set, this problem will occur. 
![1](https://github.com/user-attachments/assets/928d2bd7-fb64-4654-b6cd-a7ec3940c5e4)
![2](https://github.com/user-attachments/assets/7f0231c6-6014-4964-9823-c0754097af0e)

**Comment by andrew-platt** [comment]
Just to clarify, is this occurring with both 3.5.0 and 3.5.3?

**Reply by jjonkman** [reply]
It looks like the same issue is reported in: https://github.com/OpenFAST/openfast/discussions/2316.

**Reply by Hitszh11** [reply]
Yes, this error occurred in both 3.5.0 and 3.5.3.

**Reply by andrew-platt** [reply]
After further investigation, it is not possible to use the _LIDAR_ from _InflowWind_ at the _FAST.Farm_ level at this time.  The _LIDAR_ needs some information about the location of the turbine hub from _ElastoDyn_ and information from _ServoDyn_, however this information is not available at the _FAST.Farm_ level since this instance of _InflowWind_ is not associated with a turbine.

Usage of the _LiDAR_ with _InflowWind_ is now disabled in the upcoming 3.5.5 release with a warning.
